### PR TITLE
Tried to fix issue #4

### PR DESCRIPTION
**Pull request by HolyThunder**

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ const specialFoodColor = "#53d724";
 let tileCount = playground.width / gridSize;
 let velocityX = 0;
 let velocityY = 0;
+const oldX = 0;
+const oldY = 0;
+
```
Comment (kbkk) [inline scripts/main.js]: those should be a let, not const

**Comment by kbkk** [comment]
Please try to run the game before pull requesting

File: scripts/main.js (hunk 1 above)
Comment (HolyThunder) [reply scripts/main.js]: Riiight. Apologies.

**Comment by kbkk** [comment]
Did you try to run the game? I believe there's a missing brace somewhere from your previous pr

**Comment by HolyThunder** [comment]
Should be okay now. Sorry for the lack of testing before committing.

**Pull request merged**